src/events.json

```diff
@@ -65,10 +65,52 @@
 	},
 	"FratPartyEvent": {
 		"prompt": "You are invited to a toga party! Are you going to go?",
+		"imgPath": "FratPartyEvent/toga.png",
+		"choices": [
+			"UseWheels",
+			"SquirrelPictures"
```

Comment:
Copy and paste issue? Probably not supposed to do that. May want to check to see if any other choices are mis-labelled or mis-assigned.

Reply:
good catch

Reply:
I think this was a merge issue. Stuff got really weird with all the changes to the JSON in other branches

Reply:
which one was it? I compared every line in my last merge. maybe some commit before that?

Reply:
eh I dunno, things went weird